sharktank/sharktank/ops/qconv_impls.py

```diff
@@ -211,7 +216,7 @@ def _invoke_int32_conv2d(
             # may want to support an optional bias through the kernel.
             bias = torch.zeros((weight.shape[1],), dtype=input.dtype)
         y_qs = kernels.conv_2d_nchw_fchw(
-            input,
+            padded_input,
```

Comment:
Can't comment on unchanged lines. Please remove the `padding` function arg and remove from the FP conv2d call below.

Reply:
done